dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/Support/SimpleModelFixture.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.Extensions.VectorData;
+using VectorDataSpecificationTests.Models;
+
+namespace VectorDataSpecificationTests.Support;
+
+public abstract class SimpleModelFixture<TKey> : VectorStoreCollectionFixture<TKey, SimpleModel<TKey>>
```

Comment:
nit: RecordConformanceTests and BatchConformanceTests seems small (and similar) enough that we may want to just merge them - if we do that, then this "shared" fixture just becomes the inner class in that test type (like how BasicFilterTests has an inner Fixture). Similarly, SimpleModel also can go inside that type - it basically would just contain everything rather than having to have external shared types for these two test suites.

But no strong feels, am OK either way.

Reply:
OK yeah, that sounds like indeed a thing to cover for all connectors via the tests